tests/models/clap/test_modeling_clap.py

```diff
@@ -203,7 +206,7 @@ def check_hidden_states_output(inputs_dict, config, model_class):
 
             self.assertListEqual(
                 list(hidden_states[0].shape[-2:]),
-                [self.model_tester.patch_embeds_hidden_size, self.model_tester.patch_embeds_hidden_size],
+                [2 * self.model_tester.patch_embeds_hidden_size, 2 * self.model_tester.patch_embeds_hidden_size],
```

Comment:
This is the only change I had to do to the CLAP tests to make them pass with the tiny version. Not sure if it's work investigating more @younesbelkada